sdk/management/README.md

```diff
@@ -0,0 +1,221 @@
+# Azure management client library for Java
+
+The Azure Management Library for Java is a higher-level, object-oriented API for *managing* Azure resources,
```

Comment:
Which should be the expected for us in this repo, `Azure management client library for Java`, `Azure Management Library for Java`? Or using `libraries` as we did in previous repo? To distinguish the dataplane SDKs and auto-generated Lite SDKs.

Reply:
The title is due to this https://github.com/Azure/azure-sdk-for-java/blob/master/eng/.docsettings.yml#L16
Currently not want to touch that yet.

Given this README is located for all mgmt sdks, yes, I agree `libraries` would be better.